### ISSUE TITLE
Confusing hints on thread.gmane webpages.

### ISSUE THREAD

**Issue by Earnestly** [The-Compiler/qutebrowser#825]
This is just incase anyone else is surprised by this:
![gmane](https://ptpb.pw/w2mv.png)

I'm not entirely sure if this is fixable at all.

**Comment by Carpetsmoker** [comment]
Direct link: http://news.gmane.org/gmane.editors.vim

**Comment by The-Compiler** [comment]
The reason this happens is because `f` doesn't only hint links, but images as well - and gmane uses images to draw those trees.

I think I did that to be compatible with dwb (but I can't check as it segfaults there), but maybe the default keybindings should only hint links?

**Comment by Carpetsmoker** [comment]
Yeah, `dwb` also highlights images on this page, and has the same problem.

There's another issue on this page by the way: If you enter hint mode, and then scroll the top frame, the "new" lines don't have hints... (`dwb` also has this problem).

Also, here's another example with a similar problem (not publicly accessible):

![2015-07-21-165523_1920x1080_scrot](https://cloud.githubusercontent.com/assets/1032692/8803819/6a880ae6-2fc9-11e5-91f5-24b1da321f5d.png)

Here, the text is a link, and the `tr` row has a click event (to load the first link)...

Ideally, qutebrowser should never draw hints on top of each other...

**Comment by The-Compiler** [comment]
> There's another issue on this page by the way: If you enter hint mode, and then scroll the top frame, the "new" lines don't have hints... (dwb also has this problem).

Yeah, that's a performance thing - hints are only drawn for things visible on the page, and you can't easily know when a new link got visible. However, I think it's uncommon to scroll after starting hinting, so I don't think this is really an issue.

For the overlapping hints issue, see #71.
